ios/RCTPSPDFKit/RCTPSPDFKitViewManager.m

```diff
@@ -109,7 +109,7 @@ @implementation RCTPSPDFKitViewManager
   });
 }
 
-RCT_EXPORT_METHOD(addAnnotation:(NSString *)jsonAnnotation reactTag:(nonnull NSNumber *)reactTag) {
+RCT_EXPORT_METHOD(addAnnotation:(id)jsonAnnotation reactTag:(nonnull NSNumber *)reactTag) {
```

Comment:
This fixes an issue where they JSON annotation needed to be a string. It can now be a dictionary too.